tests/integration/test_aws/test_remove_from_bucket.py

```diff
@@ -0,0 +1,112 @@
+import os
+
+import pytest
+from wazuh_testing import global_parameters
+from wazuh_testing.modules.aws import event_monitor
+from wazuh_testing.modules.aws.s3_utils import file_exists
+from wazuh_testing.tools.configuration import (
+    get_test_cases_data,
+    load_configuration_template,
+)
+
+pytestmark = [pytest.mark.server]
+
+
+# Generic vars
+TEMPLATE_DIR = 'configuration_template'
+TEST_CASES_DIR = 'test_cases'
+MODULE = 'remove_from_bucket_test_module'
+TEST_DATA_PATH = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+CONFIGURATIONS_PATH = os.path.join(TEST_DATA_PATH, TEMPLATE_DIR, MODULE)
+TEST_CASES_PATH = os.path.join(TEST_DATA_PATH, TEST_CASES_DIR, MODULE)
+local_internal_options = {'wazuh_modules.debug': '2', 'monitord.rotate_log': '0'}
+
+# ---------------------------------------------------- TEST_REMOVE_FROM_BUCKET -----------------------------------------
+# Configuration and cases data
+configurations_path = os.path.join(CONFIGURATIONS_PATH, 'configuration_remove_from_bucket.yaml')
+cases_path = os.path.join(TEST_CASES_PATH, 'cases_remove_from_bucket.yaml')
+
+configuration_parameters, configuration_metadata, case_ids = get_test_cases_data(cases_path)
+configurations = load_configuration_template(configurations_path, configuration_parameters, configuration_metadata)
+
+
+@pytest.mark.tier(level=0)
+@pytest.mark.parametrize('configuration, metadata', zip(configurations, configuration_metadata), ids=case_ids)
+def test_remove_from_bucket(
+    configuration, metadata, upload_file_to_s3, load_wazuh_basic_configuration, set_wazuh_configuration,
+    clean_s3_cloudtrail_db, configure_local_internal_options_function, truncate_monitored_files, restart_wazuh_function,
+    wazuh_log_monitor
+):
+    """
+    description: The uploaded file was removed after the execution.
+    test_phases:
+        - setup:
+            - Load Wazuh light configuration.
+            - Apply ossec.conf configuration changes according to the configuration template and use case.
+            - Apply custom settings in local_internal_options.conf.
+            - Truncate wazuh logs.
+            - Restart wazuh-manager service to apply configuration changes.
+        - test:
+            - Check in the ossec.log that a line has appeared calling the module with correct parameters.
+            - Check that the uploaded log was removed by the module after the execution.
+        - tierdown:
+            - Truncate wazuh logs.
+            - Restore initial configuration, both ossec.conf and local_internal_options.conf.
+    wazuh_min_version: 4.5.0
+    parameters:
```

Comment:
Missing information about some fixtures: 
- `upload_file_to_s3`
- `clean_s3_cloudtrail_db`

Reply:
Fixed in https://github.com/wazuh/wazuh-qa/pull/3624/commits/ab5715bf1817d6645a055fe8c3a2f19380f7ab7c